crates/top/rerun_c/src/lib.rs

```diff
@@ -512,7 +512,7 @@ thread_local! {
 pub extern "C" fn rr_recording_stream_free(id: CRecordingStream) {
     if THREAD_LIFE_TRACKER.try_with(|_v| {}).is_ok() {
         if let Some(stream) = RECORDING_STREAMS.lock().remove(id) {
-            stream.disconnect();
+            drop(stream);
```

Comment:
I'm missing the implications of this change, if any.

Reply:
Added the note from the PR as comment.